src/pages/DatasetCatalog.js

```diff
@@ -64,6 +66,16 @@ class DatasetCatalog extends Component {
     });
   }
 
+  async getDacs() {
+    let dacs = await DAC.list(false);
+    let dacIdsAndNames = dacs.map(dac => {
+      return {id: dac.dacId, name: dac.name};
+    });
```

Comment:
Maybe this is a bug on the back end, but it doesn't filter by DACs I am a member of if I log in as a DAC Chair. It seems to be bringing in ALL DACs which is not the case. Let's file a bug for that since we can't let a chair assign a dataset to a DAC they are not in.

Reply:
I agree, that's an oversight from the backend on my part. Here's the new ticket: https://broadinstitute.atlassian.net/browse/DUOS-811

Reply:
Thank you!